packages/mobile/src/analytics/constants.ts

```diff
@@ -88,6 +88,18 @@ export enum CustomEventNames {
   invite_friends_sms = 'invite_friends_sms', // to count the # of taps on “Invite with SMS" button on Invite_Friends_Review
   invite_friends_whatsapp = 'invite_friends_whatsapp', // to count the # of taps on “Invite with WhatsApp" button on Invite_Friends_Review
 
+  backup_start = 'backup_start', // ‘set up now’ button click
+  backup_setup_info = 'backup_setup_info',
+
+  backup_quiz_backspace = 'backup_quiz_backspace', // whenever the backspace is pressed
+
+  backup_quiz_submit = 'backup_quiz_submit',
+  backup_quiz_success = 'backup_quiz_success',
+  backup_quiz_incorrect = 'backup_quiz_incorrect',
+
+  backup_setup_toggle_complete = 'backup_setup_toggle_complete',
+  backup_setup_toggle_incomplete = 'backup_setup_toggle_incomplete',
+
   // Screen name: Backup_Phrase, Backup_Insist, Backup_Share, Backup_Set
   set_backup_phrase = 'set_backup_phrase', // (count # of taps on “Set Backup Phrase” in Backup_Phrase) [we should not track the actual value of this field, just whether the user filled it out]
```

Comment:
would be great to remove all the old constants we no longer need so we dont crowd this file

Reply:
oh thats a good point.